consensus/src/detail/mod.rs

```diff
@@ -567,9 +608,7 @@ impl<'a> ConsensusRef<'a> {
     }
 
     fn check_consensus(&self, block: &Block) -> Result<(), BlockError> {
```

Comment:
This function can be eliminated. No need for one liners in this context.

Reply:
Done.